quickstarts/mongodb-prometheus/config.yml

```diff
@@ -0,0 +1,67 @@
+id:
+
+# Name of the quickstart
+slug: mongodb-prometheus
+
+# Title of the quickstart
+title: MongoDB (Prometheus)
+
+# Description of the quickstart
+description: |
+  ## MongoDB Monitoring
+
+  MongoDB enables the unlimited virtual scaling of applications. Utilities like mongostat and mongotop offer immediate results but fail to provide insights into trends in a highly graphical visual dashboard. MongoDB dashboards provide insights into key metrics like RAM usage, operations per second, page fault, disk size, lock %, and app and database performance at a glance. 
```

Comment:
..... key metrics like connect saturation, oplog size, query and document operations, databases and indexes size, collections and transactions latency, replica set and members status.

Reply:
Done